### PR TITLE
[Yokogawa End of STEP9] Set timezone and customize error page

### PR DESCRIPTION
**Pull request by Take-st**
## Overview
-  Finished end of `STEP9 configuring rails application` of the [Rails training](https://github.com/Fablic/training/blob/develop/steps_jp.md)

## What I've DONE
- Set Tokyo timezone
- Customized error pages
  - ~~But I didn't change the setting [here]~~(https://github.com/Fablic/training/blob/70fce8f1dce5bcf976ff90c779dafe2230eb55ec/myapp/config/environments/development.rb#L15).
  - ~~Just because it will get harder to debug if I set that false and hide full error reports.~~
  - [updated] I changed that to false.
- Remove apple-touch-icon.png and apple-touch-icon-precomposed.png
  - Because it seems like they are crashed and not needed.
  - [FYI] They are used for the short icon for iOS device, but I don't assume this application is used by iOS devices.
- Update favicon.ico

## Supliment
- capture of error pages
  - image
  - image

### CODE REVIEW

File: myapp/config/application.rb [hunk 1]
```diff
@@ -16,7 +16,8 @@ class Application < Rails::Application
     # These settings can be overridden in specific environments using the files
     # in config/environments, which are processed later.
     #
-    # config.time_zone = "Central Time (US & Canada)"
+    config.time_zone = 'Tokyo'
+    config.active_record.default_timezone= :local
```
Comment (Take-st) [inline myapp/config/application.rb:20]: ref: https://zenn.dev/ryouzi/articles/dda18594f2dbd3

File: myapp/docker-compose.yml [hunk 2]
```diff
@@ -28,4 +28,5 @@ services:
       DB_NAME: myapp
       DB_PASSWORD: password
       DB_HOST: db
+      TZ: Asia/Tokyo
```
Comment (Take-st) [inline myapp/docker-compose.yml:31]: - setting for OS in docker

File: myapp/config/application.rb [hunk 3]
```diff
@@ -16,7 +16,8 @@ class Application < Rails::Application
     # These settings can be overridden in specific environments using the files
     # in config/environments, which are processed later.
     #
-    # config.time_zone = "Central Time (US & Canada)"
+    config.time_zone = 'Tokyo'
```
Comment (Take-st) [inline myapp/config/application.rb:19]: - setting for rails application

File: myapp/config/application.rb (hunk 1 above)
Comment (Take-st) [inline myapp/config/application.rb:20]: - setting for ActiveRecord

**Comment by h-aqua25** [comment]
I wanna look at the 404, 500 screen with local path.

**Comment by Take-st** [comment]
@h-aqua25 Thank you for your review. I updated the capture images in the Pr description. Could you check it again?

File: myapp/app/controllers/tasks_controller.rb [hunk 4]
```diff
@@ -49,7 +49,7 @@ def destroy
   private
 
   def fetch_task_by_params_id
-    @task = Task.find_by(id: params[:id])
+    @task = Task.find(params[:id])
```
Comment (Take-st) [inline myapp/app/controllers/tasks_controller.rb:52]: - Because it doesn't throw Exception when I use find_by.

**Comment by Take-st** [comment]
- I saw your PR and found that [you added these lines](https://github.com/Fablic/training/pull/1411/files#diff-ab27f810b8b9c15ac56a2cb47cfb658f388019a9643fe6397f423ebf565209f5R4-R6). But it doesn't make sense simply because it doesn't work well. So I don't implement that code.
  - Rails automatically shows 404 page when it catches ActiveRecord::RecordNotFound and ActionController::RoutingError
  - And, Since ActionController::RoutingError is thrown before reaching ApplicationController, `rescue_from ActionController::RoutingError, with: :render_404` will never work.
  - Rails automatically shows 500 page when it catches Exception

see: https://qiita.com/mr-myself/items/c2f4fb2e5dcee6a336f3#comment-23298b703d75b7d27487

**Comment by Take-st** [comment]
@h-aqua25 I added a comment! Could you kindly check it again? If you need help, contact me in person!

**Review by h-aqua25 — APPROVED** [review]
LGTM!
LGTM

**Pull request merged**